<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="historique">
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>CIN:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cin_aj"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>id d'offre:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_O"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Centre:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_C"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="type">
              <item>
               <property name="text">
                <string>en ligne</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>presentiel</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Dates :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QDateEdit" name="date_Aj"/>
          </item>
          <item row="3" column="0">
           <widget class="QCalendarWidget" name="calendrier"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Heures:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QTimeEdit" name="time_aj"/>
          </item>
          <item row="6" column="0">
           <layout class="QFormLayout" name="formLayout_7"/>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="ajouter">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <widget class="QWidget" name="gridWidget" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>141</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="1">
             <widget class="QLineEdit" name="offre_A"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>ID Offre:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>149</x>
             <y>9</y>
             <width>561</width>
             <height>341</height>
            </rect>
           </property>
           <property name="title">
            <string>Entretiens</string>
           </property>
           <widget class="QTableView" name="table_e">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>541</width>
              <height>311</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>141</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>    Num:   </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="num_A"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="modifier">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="6" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Date :</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDateEdit" name="date_M"/>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>heures:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QTimeEdit" name="time_M"/>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_M">
            <item>
             <property name="text">
              <string>en ligne</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>presentiel</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="num_M"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Num:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="num_r">
            <property name="text">
             <string>Recherche</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="Supprimer">
          <property name="text">
           <string>supprimer</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox_S"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Num:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tri date</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tri"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Confirmation date</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>email:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Email_C"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="mail">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;Bonjour Mr\Mm,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;	Je vous confirme la date de votre entretien prévue le &amp;quot;&amp;quot; avec l'entreprise &amp;quot;&amp;quot; aura lieu dans deux jours . &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;	Soyer à l'heure et bonne chance à vous  .&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;				&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;				&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Statistiques</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>ID ENTREPRISE:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxStat"/>
          </item>
          <item row="2" column="1">
           <widget class="QProgressBar" name="progressBar_2">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Entretien Accepté:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Entretien Réfusé:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Historiques</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QTableView" name="view_histo"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendrier</sender>
   <signal>clicked(QDate)</signal>
   <receiver>date_Aj</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
